samples/Samples.Authorization/Program.cs

```diff
@@ -1,6 +1,5 @@
 using AuthorizationSample.Data;
 using AuthorizationSample.Schema;
-using GraphQL;
```

Comment:
This is a constant pain point on our GitHub issues board -- people not understanding what `using` statements are required -- I feel like I've answered this question a half dozen times at least.  I do not think we should simplify the using statements for these statements for that reason.

Of course I would put the using statement into the csproj or Directory.Build.props.  For the main project here you'll see I added enough global usings that there are very few cases needed otherwise.  Similarly for tests there are many global usings.  However, these are samples intended for users to most easily be able to copy and paste and/or compare to their own code.  We should leave the explicit using statements.

Reply:
(ditto elsewhere in the samples)

Reply:
Alternatively we can update the readme to suggest putting the using statement into the csproj.  I wouldn't do so probably, but I'm okay with that option if you want to.  Then we should move it back out of the samples props file into the csproj of each sample so the samples can run "standalone".

Reply:
Another thing we may want to consider is specifically having the samples props file NOT inherit from the main one, so that we can be sure there are no settings being inherited unintentionally, so that the sample projects are "vanilla".  Then we only add stuff to the samples props file that we specifically want for testing purposes, like InternalsVisibleTo or enabling the analyzers.  Just an idea.

Reply:
I added usings back.